misc/specs/atrium-specs-jvm/build.gradle

```
@@ -16,6 +16,7 @@ dependencies {
     // exclude this dependency in case you want to use another implementation
     runtimeOnly prefixedProject('domain-robstoll-jvm')
     runtimeOnly prefixedProject('core-robstoll-jvm')
+    runtimeOnly prefixedProject('domain-robstoll-kotlin_1_3-common')
```

```suggestion
    runtimeOnly prefixedProject('domain-robstoll-kotlin_1_3-jvm')
```
+ you need to add it also to atrium-specs-android